challenges/objects-arrays.js

```diff
@@ -45,7 +67,11 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
+let universities = [];
+for (let i = 0; i < graduates.length; i++) {
+  universities.push(graduates[i]["university"]);
+}
+universities = universities.sort();
```

Comment:
you can keep universities as const, and then on line 74 just do universities.sort(). Discussed further during 1:1